util/archutil/fixtures/exit.ppc64.s

```diff
@@ -0,0 +1,10 @@
+	.global _start
+	.section ".opd","aw"
+_start:		
+	.quad .L.start,.TOC.@tocbase,0
+	.text
+	.abiversion 1
+.L.start:
+	li %r0, 234
```

Comment:
`exit_group` works fine but leaves one wondering why simpler `exit` was not used instead

Reply:
I just used whatever glibc _exit() used.  :-)  I'll change it to __NR_exit (1) as that will probably work better with really old kernels.